grapher/pydepwrap.py

```diff
@@ -17,12 +17,30 @@
 def setup_dirs(rootdir: str) -> List[str]:
     return pydep.setup_py.setup_dirs(rootdir)
 
-
-def setup_info_dir(setup_dir: str) -> Any:
-    info, err = pydep.setup_py.setup_info_dir(setup_dir)
+# pydep_in_python2 uses Python2 to run pydep and returns parsed JSON object.
+def pydep_in_python2(cmd: str, dir: str) -> Any:
+    rootdir = os.path.dirname(os.path.dirname(__file__))
+    if os.environ.get('SRCLIBPATH') is not None: # Not None only in Docker container
```

Comment:
> Is this necessary for running inside a Docker container? 

No... just Docker container for srclib-python has this env var set and we could use it. I'll remove the comment here.